code/prime-number/Model/src/test/java/ru/unn/agile/primenumber/model/PrimeNumberFinderTest.java

```diff
@@ -0,0 +1,117 @@
+package ru.unn.agile.primenumber.model;
+
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+public class PrimeNumberFinderTest {
+
+    @Test
+    public void canInitFinder() {
+        var finder = new PrimeNumberFinder(0, 10);
+        assertNotNull(finder);
+    }
+
+    @Test(expected = IllegalArgumentException.class)
+    public void cannotInitFinderWithNegativeStart() {
+        var finder = new PrimeNumberFinder(-1, 10);
+    }
+
+    @Test(expected = IllegalArgumentException.class)
+    public void cannotInitFinderWithNegativeEnd() {
+        var finder = new PrimeNumberFinder(0, -1);
+    }
+
+    @Test(expected = IllegalArgumentException.class)
+    public void cannotInitFinderWithEndBiggerThanStart() {
+        var finder = new PrimeNumberFinder(10, 0);
+    }
+
+    @Test
+    public void cannotFindPrimeNumberBefore2() {
+        var finder = new PrimeNumberFinder(0, 2);
+        var result = finder.findNumbers();
+        List<Integer> expectedResult = new ArrayList<>();
```

Comment:
Why Integer instead int?
Can the list contain null ?

Reply:
https://stackoverflow.com/questions/18021218/create-a-list-of-primitive-int

Reply:
Yep

Reply:
Can the list contain null ?
----
Yes, and what? 
My method findNumbers that creates results don't put null in list, it is impossible situation, that we will get NullPointerException there, if that's what you're getting at.

Reply:
It was just a question :)